app/subscription_handlers/work_accessioning_completed_subscription_mailer.rb

```diff
@@ -14,14 +14,33 @@ def initialize(object:)
   def call
     return unless object.is_a?(Work)
 
-    if object.first_version?
-      WorksMailer.with(work: object).deposited_email.deliver_later
-    else
-      WorksMailer.with(work: object).new_version_deposited_email.deliver_later
+    users = [object.user]
+    object.shares.deposit.each do |share|
+      users << share.user
+    end
```

Comment:
Are lines 17-20 used/needed based on lines 22-24?

Reply:
Half refactor. Thanks for catching.